src/freedv_api.c

```diff
@@ -1372,6 +1372,28 @@ int freedv_get_bits_per_modem_frame       (struct freedv *f) {return f->bits_per
 int freedv_get_rx_status                  (struct freedv *f) {return f->rx_status;}
 void freedv_get_fsk_S_and_N               (struct freedv *f, float *S, float *N) { *S = f->fsk_S[0]; *N = f->fsk_N[0]; }
 
+
+/*---------------------------------------------------------------------------*\
+
+  FUNCTIONS...: freedv_set_fmin_fmax
+  AUTHOR......: Simon Lang - DJ2LS
```

Comment:
Do we actually need to set both fmin and fmax?  Maybe one argument is enough.  Then again we would need to make it clear that the total acquisition range is twice the single argument :thinking: 

Re naming how about:
`freedv_set_tuning_range(fmin, fmax)` or `freedv_set_freq_acquisition_range(fmin, fmax)`

Reply:
My thoughts with using both - fmin AND fmax are, that it could be interesting, keeping this settings flexible. Maybe if two stations need to run near together, it could be interesting, if the tuning to the next station is "less sensitive / less wide" so they are not affecting each other. Maybe its like a "filter" or "out filtering" traffic of the neighbor station. But this are just theoretical thoughts for being flexible for possible future use-cases 🤔 

Yes, renaming the function to` freedv_set_tuning_range(fmin, fmax)` sounds good.

Reply:
Using `fmin` and `fmax` also makes the entire range (e.g. -100 to +100, 200Hz total) explicit, so I agree - lets keep both arguments :+1:

Reply:
I also did tests with unequal fmin-fmax values like  fmin = -150 and fmax = 50, which where successful. Or shouldn't they?

Reply:
If we're allowing unequal values, then yes, having it work successfully is expected. 👍

Reply:
Is this still going to be renamed to freedv_set_tuning_range?

Reply:
@tmiw this should already be solved by 
https://github.com/drowe67/codec2/commit/a45310aff1402d0bd86da31c4fd2d3812156b283

Reply:
Yep, for some reason I had an older version displayed.